jina/clients/python/__init__.py

```diff
@@ -70,7 +70,7 @@ def check_input(input_fn: Union[Iterator['jina_pb2.Document'], Iterator[bytes],
         :param input_fn: the input function
         """
         if hasattr(input_fn, '__call__'):
-            input_fn = input_fn()
+            input_fn = input_fn()  # type: ignore
```

Comment:
remove `type ignore` (mypy is just a tool to help us but not our purpose to mitigate any metric around it)